application/defs/include/osib_example.yml

```
@@ -0,0 +1,361 @@
+####################################################################################
+# osib_short.yaml: Open Security Information Base (OBIB), (c) OWASP Top 10 team    #
+# This work is licendes under a Creative Commons Attribution 3.0 Unported Licence  #
+# ---------------------------------------------------------------------------------#
+# Example File                                                                     #
+####################################################################################
+
+doctype:        OSIB
+schema:         90         # V 0.90
+date:           20211104
+children:
+  2:
+    alias:      "owasp"
+  "owasp":
+    attributes:
+      sources_i18n:
+        en:
+          name:             "Open Web Application Security Project&reg; (OWASP)"
+    children:
+      1:
```

can you remind me why we need numbered children? could we instead force the aliases to be unique ids or names?
```suggestion
      - id: 1
 ```
or
```suggestion
      - name: 1
```

---

Yes, we don't need numbers for the projects, standards or tools. They were there for historical reasons or if we like to define a stable 'Id' for a context root. Names can change...
Other possible solution: use the 'alias' token to accept changing names for projects, standards or tools and to provide backward compatibility...

---

We will need numbers inside the project's, standard's or tool's subtrees as theiry table of contents use then usually to identify chapters, sections and subsections.